modules/appeals_api/app/workers/appeals_api/supplemental_claim_upload_status_updater.rb

```diff
@@ -15,6 +15,7 @@ def perform(ids)
       batch_size = CentralMailUpdater::MAX_UUIDS_PER_REQUEST
       SupplementalClaim.where(id: ids).find_in_batches(batch_size: batch_size) do |batch|
         CentralMailUpdater.new.call(batch)
+        sleep 5 # Avoid flooding CMP with requests all at once
```

Comment:
So the Batch job actually only sends in 100 IDs into this job, which is the same as the MAX_UUIDS_PER_REQUEST. The chances of this ever running more than once are basically zero, but I wanted to keep the batch code around just in case something else changed (like MAX_UUIDS_PER_REQUEST changes to 50), and to keep it obvious that there is a limit on how many IDs CMP will respond to per request.

The point of the sleep is so that each batch of `batch_size` IDs will not be sent at "the same instant" to the CMP services - exponential backoff of the job won't help with flooding CMP with requests if the batch loop runs 5 times during the `perform`.

Reply:
A 5 second delay could help us..or it could hurt us right ?  Delaying OUR execution by 5 seconds could cause our call to collide with another caller of the service...and we still end up with timeouts from the service......

Reply:
It could, but that's already the case regardless of whether we delay or not. I'll go ahead an make a quick change regardless, since as written this will always stall our run by 5 seconds even if we only run once in the batch loop.

Reply:
Something that would help the service itself, however, is reducing it's load by further sidekiqing these batches out..the way it is done in this PR, if one of the batches times out, we will call the service to 11 times with all the original IDs, even if some of those IDs were in batches that did not time out and they have the latest status...however, if we sidekiq  here, like I'm suggesting, we could take advantage of sidekiq to re-run only the batches(ids) that timed out....

Reply:
The `find_in_batches` in this job should only ever run through its loop once. We call this job's `perform` from `HigherLevelReviewUploadStatusBatch` with 100 IDs (sometimes multiple times w/ a timing offset) and this job's UNIQUE_IDS_PER_REQUEST is also 100. This job is already doing what you suggest, it's just super-CYA by being wrapped in a `find_in_batches` loop in case UNIQUE_IDS_PER_REQUEST changes out from under us. Otherwise (and in all current cases) it will only request CMP once with 100 IDs.

Besides, I'm not a fan of a job that creates jobs that creates jobs :) I'd say we should rethink our approach if that's what it ends up trying to do.

Reply:
I'm going to drop this, not a big deal since we are not going to hit multiple calls here, and the real issue is the scheduler creating duplicate jobs...